configs.py

```diff
@@ -1,11 +1,11 @@
 import datetime
 
 
-
 is_local_run = False
 is_min_max_norm = True
 run_from_sample_data = False
-sample_args = ['main.py', 'feature_engineering_lstm', 'all']
+run_gpu = False
+sample_args = ['main.py', 'train_process', '0', 'all']

```

Comment:
This is for when you are running a GPU server but you are not actually using the GPU for Tensorflow. I had an error about saving the Tensorflow model after using multi GPU cores. that is why added in order to make it optional